framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -525,11 +526,13 @@ private void applyDefaultsElement(final Element annotationScope, final Annotated
                 // but it doesn't work correctly and tests fail.
```

Comment:
The comment about "last three clauses" is out of date, or at least confusing, on the line before this one.